install/installer/pkg/components/agent-smith/configmap.go

```diff
@@ -21,25 +19,21 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 	ascfg := config.ServiceConfig{
 		PProfAddr:      fmt.Sprintf("localhost:%d", PProfPort),
 		PrometheusAddr: fmt.Sprintf("localhost:%d", PrometheusPort),
+		Namespace:      ctx.Namespace,
 		Config: config.Config{
-			Blocklists: &config.Blocklists{
-				Very: &config.PerLevelBlocklist{
-					Signatures: []*classifier.Signature{{
-						Name:    "testtarget",
-						Domain:  classifier.DomainProcess,
-						Kind:    classifier.ObjectELFSymbols,
-						Pattern: []byte(base64.StdEncoding.EncodeToString([]byte("agentSmithTestTarget"))),
-						Regexp:  false,
-					}},
-				},
-			},
-			Kubernetes: config.Kubernetes{Enabled: true},
+			Kubernetes:          config.Kubernetes{Enabled: true},
+			KubernetesNamespace: ctx.Namespace,
 			GitpodAPI: config.GitpodAPI{
 				HostURL: fmt.Sprintf("https://%s", ctx.Config.Domain),
 			},
 		},
 	}
 
+	if ctx.Config.Experimental.AgentSmith != nil {
+		ascfg.Config = *ctx.Config.Experimental.AgentSmith
+		ascfg.Config.KubernetesNamespace = ctx.Namespace
+	}
```

Comment:
```suggestion
	_ = ctx.WithExperimental(func(cfg *experimental.Config) error {
		if cfg.AgentSmith != nil {
			ascfg.Config = *cfg.AgentSmith
			ascfg.Config.KubernetesNamespace = ctx.Namespace
		}
		return nil
	})
```

Reply:
Doh! You are right, should have done it myself. Thanks for catching this. Fixed!